lib/dish.rb

```diff
@@ -0,0 +1,19 @@
+
+class Dish
```

Comment:
This is so different to what I've done! You've taken an interesting approach with the different classes :)